apps/omg_watcher/test/challenger/core_test.exs

```diff
@@ -42,57 +42,40 @@ defmodule OMG.Watcher.Challenger.CoreTest do
         txindex2: 0,
         oindex2: 1,
         cur12: <<0::160>>,
-        newowner1: "alice",
+        newowner1: <<0::160>>,
         amount1: amount1,
-        newowner2: "bob",
+        newowner2: <<1::160>>,
         amount2: amount2
       },
-      sig1: "sig1",
-      sig2: "sig2"
+      sig1: <<0::520>>,
+      sig2: <<0::520>>
     }
 
-    txid = Signed.signed_hash(signed)
+    txhash = Signed.signed_hash(signed)
 
     %TransactionDB{
-      blknum1: 1,
-      txindex1: 0,
-      oindex1: 0,
-      blknum2: 1,
-      txindex2: 0,
-      oindex2: 1,
-      cur12: <<0::160>>,
-      newowner1: "",
-      amount1: amount1,
-      newowner2: "",
-      amount2: amount2,
-      txblknum: 2,
+      blknum: 2,
       txindex: txindex,
-      txid: txid,
-      sig1: "sig1",
-      sig2: "sig2"
+      txhash: txhash,
+      inputs: [
+        %TxOutputDB{creating_tx_oindex: 0, spending_tx_oindex: 0}
+      ],
+      outputs: [
+        %TxOutputDB{creating_tx_oindex: 0},
+        %TxOutputDB{creating_tx_oindex: 1}
+      ],
+      txbytes: Signed.encode(signed)
     }
   end
 
   @tag fixtures: [:transactions]
   test "creates a challenge for an exit", %{transactions: transactions} do
-    utxo_exit = Utxo.position(1, 0, 0)
     challenging_tx = hd(transactions)
 
     expected_cutxopos = Utxo.position(2, 1, 0) |> Utxo.Position.encode()
 
-    %Challenge{cutxopos: ^expected_cutxopos, eutxoindex: 0} =
-      Core.create_challenge(challenging_tx, transactions, utxo_exit)
+    assert %Challenge{cutxopos: ^expected_cutxopos, eutxoindex: 0} = Core.create_challenge(challenging_tx, transactions)
 
-    [_, challenging_tx | _] = transactions
-
-    expected_cutxopos = Utxo.position(2, 2, 1) |> Utxo.Position.encode()
-
-    %Challenge{cutxopos: ^expected_cutxopos, eutxoindex: 0} =
-      Core.create_challenge(challenging_tx, transactions, utxo_exit)
-
-    utxo_exit = Utxo.position(1, 0, 1)
-
-    %Challenge{cutxopos: ^expected_cutxopos, eutxoindex: 1} =
-      Core.create_challenge(challenging_tx, transactions, utxo_exit)
+    # TODO: consider what should be tested here
```

Comment:
I'm not fully followed the purpose of this test... anyway remain open to suggestions